microsoft-edge/webview2/how-to/winrt-js-conversion.md

```diff
@@ -36,6 +36,10 @@ The WebView2 WinRT JS Projection tool (**wv2winrt**) converts from WinRT to Java
 | Class static member | JavaScript object property | See below. |
 | Class constructor | JavaScript constructor and function | See below. |
 
+A couple notes for passing JavaScript objects to host objects:
+*  If JavaScript Date objects need to be passed to host objects as `VT_DATE`, the host objects property `shouldSerializeDates` should be set to true. By default, it is passed to host as string using `JSON.stringify`.
```

Comment:
```suggestion
*  If JavaScript `Date` objects need to be passed to host objects as `VT_DATE`, the host object's `shouldSerializeDates` property should be set to `true`. By default, `Date` objects are passed to the host as `string`, by using `JSON.stringify`.
```
* it - check whether "it" means Date objects.  prefer being specific instead of "it".